platform/lib/routers/thumbor.js

```diff
@@ -62,7 +62,6 @@ thumborRouter.get(imagePaths, (request, response, next) => {
     // rewrite the URL back to the original one
     if (error.code == 'ECONNREFUSED') {
       request.url = imageUrl.href;
-      log.error('fail', request.url, error);
```

Comment:
Yes and no. My fear is logging here might confuse new contributors to amp.dev as this would log an error for every image request during local development as long as the thumbor container is not running. Ways to go about this I see: fence the log with `config.isDevMode()`, enable/disable thumbor via environment config flag (similar to how it's done for redis) and only log if enabled, check if the thumbor server is available only once (on app startup or with the first request), then stop logging. Couldn't decide and therefore only removed it 😅

Reply:
I think simply checking for devMode in the middleware is probably easiest.